heroic-test/src/main/java/com/spotify/heroic/test/AbstractMetadataBackendIT.java

```diff
@@ -378,7 +378,7 @@ public void filterTest() throws Exception {
     private void retrySome(final ThrowingRunnable action) throws Exception {
         AssertionError error = null;
 
-        for (int i = 0; i < 10; i++) {
+        for (int i = 0; i < 50; i++) {
```

Comment:
Note for reviewers: this was timing out in CI so I increased. This won't be any slower, it's just more permissive.